app/views/layouts/application.html.erb

```diff
@@ -3,13 +3,20 @@
   <head>
     <title>BudgetApp</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=PT+Serif&family=Raleway:wght@300;700&family=Source+Sans+Pro:wght@300&display=swap" rel="stylesheet">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
   </head>
 
   <body>
+    <%= render partial: "layouts/header" %>
+    <%= render partial: "layouts/flash" %>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
     <%= yield %>
```

Comment:
- In the deployed version of our application on Heroku, It would be nice you check the UI alert and notification to ensured duplicate flashes are not displayed to users, We don't want hiring managers seeing such bug, what do you think? 🤔 

![Screen Shot 2022-03-19 at 7 10 42 AM](https://user-images.githubusercontent.com/50230297/159109890-03a46c30-d7a5-4710-8bd0-e8100237260b.png)